gp-address-autocomplete/gpaa-store-static-map-entry-meta.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Gravity Perks // Address Autocomplete // Store Static Google Maps in Entry Meta
+ * https://gravitywiz.com/documentation/gravity-forms-address-autocomplete/
+ *
+ * Store the Map generated on Map Field as a Static Google Map in Entry Meta.
+ *
+ * Plugin Name:  GP Address Autocomplete
+ * Plugin URI:   https://gravitywiz.com/documentation/gravity-forms-address-autocomplete/
+ * Description:  Store the Map generated on Map Field as a Static Google Map in Entry Meta.
+ * Author:       Gravity Wiz
+ * Version:      0.1
+ * Author URI:   https://gravitywiz.com
+ */
+add_action( 'gform_after_submission', function ( $entry, $form ) {
+
+	// Target Form ID, and the Address (not Map) Field ID.
+	$form_id  = '310';
+	$field_id = '3';
+
+	// Values for Static Map Setting
+	$zoom    = 13;
+	$width   = 640;
+	$height  = 480;
+	$api_key = 'Enter your Google Maps API Key';
+
+	if ( $form['id'] != $form_id ) {
+		return;
+	}
+
+	$latitude  = $entry[ 'gpaa_lat_' . $field_id ];
+	$longitude = $entry[ 'gpaa_lng_' . $field_id ];
+	$url       = "https://maps.googleapis.com/maps/api/staticmap?center={$latitude},{$longitude}&zoom={$zoom}&size={$width}x{$height}&maptype=roadmap&key={$api_key}";
+
+	gform_update_meta( $entry['id'], 'gppa_' . $field_id, $url );
```

Comment:
Curious, why this meta key? Should it be slightly more specific such as `'gppa_' . $field_id . '_map'`?

Also, is it possible to display this in the entry detail view?

Reply:
Updated customer with a class based gist snippet, which includes adding the entry meta box. Perhaps we can push that as an overall update here on snippet-library after customer affirms?